File: Bots/Python/Skills/CodeFirst/WaterfallSkillBot/dialogs/proactive/continuation_parameters.py
# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License.

from botbuilder.schema import ConversationReference


class ContinuationParameters:
    def __init__(
        self,
        claims_dentity: str,
        oauth_scope: str,
        conversation_reference: ConversationReference,
    ):
        self.claims_dentity = claims_dentity
        self.oauth_scope = oauth_scope
        self.conversation_reference = conversation_reference

File: Bots/DotNet/Skills/Composer/ComposerSkillBotDotNet/ComposerSkillBotDotNet/ContinuationParameters.cs
using System.Security.Principal;
using Microsoft.Bot.Schema;

namespace ComposerSkillBotDotNet
{
    public class ContinuationParameters
    {
        public IIdentity ClaimsIdentity { get; set; }

        public string OAuthScope { get; set; }

        public ConversationReference ConversationReference { get; set; }
    }
}
